cs/src/Contracts/Tunnel.cs

```diff
@@ -138,4 +138,10 @@ public Tunnel()
     /// </summary>
     [JsonIgnore(Condition = JsonIgnoreCondition.WhenWritingNull)]
     public DateTime? Expiration { get; set; }
+
+    /// <summary>
+    /// Gets or the custom amount of time the tunnel will be valid if it is not used or updated.
```

Comment:
What time units? E.g. days, seconds, milliseconds?

Reply:
Seconds! Good catch, I can update the comment to reflect that